lib/osf-components/addon/components/delete-button/template.hbs

```diff
@@ -1,36 +1,38 @@
-{{#if small}}
-    {{#bs-button data-test-delete-button
-        (html-attributes aria-label=this.buttonLabel)
-        type='link'
-        size='sm'
+{{#if this.small}}
+    <BsButton 
+        data-test-delete-button
+        aria-label={{this.buttonLabel}}
+        @type='link'
+        @size='sm'
         local-class='SmallDelete'
-        onClick=(action this._show)
-        disabled=this.disabled
-    }}
+        @onClick={{action this._show}}
+        @disabled={{this.disabled}}
+    >
         {{fa-icon 'times' size='lg'}}
-    {{/bs-button}}
+    </BsButton>
 {{else}}
-    {{#bs-button data-test-delete-button
-        type='default'
-        onClick=(action this._show)
-        disabled=this.disabled
-    }}
+    <BsButton 
+        data-test-delete-button
+        @type='default'
+        @onClick={{action this._show}}
+        @disabled={{this.disabled}}
+    >
         {{this.buttonLabel}}
-    {{/bs-button}}
+    </BsButton>
 {{/if}}
 
 {{#if this.modalShown}}
-    {{#bs-modal
-        tagName='span'
-        onHide=(action this._cancel)
+    <BsModal
+        @tagName='span'
+        @onHide={{action this._cancel}}
         as |modal|
-    }}
-        {{#modal.header}}
+    >
+        <modal.header>
             <h3 class='modal-title'>{{this.modalTitle}}</h3>
-        {{/modal.header}}
+        </modal.header>
 
-        {{#modal.body}}
-            {{#if hasBlock}}
+        <modal.body>
+            {{#if @hasBlock}}
```

Comment:
`hasBlock` is actually a helper (well, it's [transformed into a helper](https://github.com/emberjs/ember.js/blob/master/packages/ember-template-compiler/lib/plugins/transform-has-block-syntax.ts)).

Reply:
Huh. So noted. So I should tell the template linter to ignore that line?

Reply:
The [current guides say](https://guides.emberjs.com/release/components/block-params/#toc_supporting-both-block-and-inline-component-usage-in-one-template) to use `has-block` and make no mention of `hasBlock`. I think it's an old way of doing this that they're still supporting. The 1.x guides [mention it](https://guides.emberjs.com/v1.13.0/components/composing-components/). We should probably switch to `has-block`.